### ISSUE TITLE
[hebits]  (Config)

### ISSUE THREAD

**Issue by liormaron26** [Jackett/Jackett#3941]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue. Duplicating issues makes it more difficult for everyone to follow. Your issue may have already been solved in the past as well.

Provide a description of the feature request or bug, the more details the better.
If you are experiencing an issue with a tracker, a **full enhanced log must be included**. Instructions for obtaining logs are here: https://github.com/Jackett/Jackett#troubleshooting

**Jackett version**:  0.10.267.0
**Mono version** (if not using Windows):  
When adding indexer "Hebits" I receive an error message: "An error occured while updating this indexer"
I checked the user name and password many times and it's OK.
I uninstalled Jackett and installed fresh again bit it doesn't help.
Please assist.

-------------------------------

**Comment by garfield69** [comment]
Please provide the **full enhanced log** file. 
1. scroll down to the bottom of the Jackett Dashboard and *tick* the *enhanced Logging* checkbox
2. scroll up a bit and *click* on the *apply server settings* button
3. repeat the test
4. find the *log.txt* file (linux `~/.config/Jackett/`, windows `%ProgramData%\Jackett`)
5. edit it with a plain text editor, redacting any personal details, usernames, passwords, passkeys, hashes etc.
6. save, then drag-drop it here for us to take a look at.
Thanks.

**Comment by liormaron26** [comment]
[log.txt](https://github.com/Jackett/Jackett/files/2473172/log.txt)

**Comment by garfield69** [comment]
@kaso17  OR @rotem925  could you please check that you can still login to hebits via Jackett, and/or take a look at this ticket please? thanks.

**Comment by rotem925** [comment]
@garfield69 seems like it works fine. Tested using the test button and did a search from hebits and got some results.. Hope that was enough. Thanks!

**Comment by garfield69** [comment]
@rotem925  thanks for checking.
@kaso17  I'm going to need your expertise on this one. Thanks.

**Comment by kaso17** [comment]
@liormaron26 it's failing because of a wrong username/password.
Are there any special/Hebrew characters in your username/password?

**Comment by liormaron26** [comment]
No, there are no any Hebrew letters. I checked the username and password.
